content/api/recipient-lists.apib

```diff
@@ -650,3 +615,45 @@ list is needed again, the list must be resubmitted with the CREATE API.
                 }
             ]
         }
+
+
+### List all Recipient Lists [GET /recipient-lists]
+
+List a summary of all recipient lists. The recipients for each list are not included in the
```

Comment:
"Returns an array abbreviated recipient list objects. The recipients for each list are not included in the results. Retrieve a single list to get a list the recipients. (link this one to the method)"